README.md

```diff
@@ -1,7 +1,8 @@
 # Vuetify Date Range Picker
 
 > The missing date range picker for Vuetify JS you have been looking for.
-![Date Range Picker](./public/updated-date-range-lres.gif)
+> ![Date Range Picker](https://github.com/praveenpuglia/vuetify-daterange-picker/blob/master/public/updated-date-range-lres.gif)
```

Comment:
We are not publishing `public` folder to npm(see `files` field in `package.json` so I removed the relative link and added GitHub URL for `.gif`)

Reply:
Awesome. Thanks 👍